gapic/templates/%namespace/%name_%version/%sub/services/%service/transports/rest.py.j2

```diff
@@ -121,15 +121,19 @@ class {{ service.name }}RestInterceptor:
     {% endfor %}
 
     {% for name, signature in api.mixin_api_signatures.items() %}
-    def pre_{{ name|snake_case }}(self, request: {{signature.request_type}}, metadata: Sequence[Tuple[str, str]]) -> {{signature.response_type}}:
+    def pre_{{ name|snake_case }}(
+        self, request: {{signature.request_type}}, metadata: Sequence[Tuple[str, str]]
+    ) -> Tuple[{{signature.request_type}}, Sequence[Tuple[str, str]]]:
```

Comment:
This doesn't look right. What is this tuple ?

Reply:
The tuple is the object returned. A few lines down we have `return request, metadata`